src/NUnitFramework/tests/Internal/Filters/NotFilterTests.cs

```diff
@@ -54,7 +54,7 @@ public void PassTest()
         {
             var filter = new NotFilter(new CategoryFilter("Dummy"));
 
-            Assert.False(filter.Pass(_topLevelSuite));
+            Assert.True(filter.Pass(_topLevelSuite));
             Assert.False(filter.Pass(_dummyFixture));
             Assert.False(filter.Pass(_dummyFixture.Tests[0]));
```

Comment:
This series of asserts correctly tests a dummy fixture containing one excluded test only. We should have another test that applies to a fixture one excluded and one non-excluded test, in order to match the problem we are trying to fix. Such a fixture should pass and it should contain one test that passes and another that fails.